components/login/EmailInputWIthLabel.js

```diff
@@ -1,9 +1,21 @@
-const EmailInputWithLabel = () => (
-    <div>
-        <input type="text" id="txtEmail" value={userEmail}
-            onChange={({ target }) => setUseremail(target.value)} />
-        <label>Email</label>
-    </div>
-);
+const EmailInputWithLabel = (props) => {
```

Comment:
Could you use props destructuring instead of `props` for component props? 1. It is way easier for collaborators to see what props the component are supposed to have without having to get into your code at length to figure it out. 2.  I've spelled out the `props` in all the components that I've coded by destructuring and think consistency is at issue as well. If you rather have me revise other components or leave them as in two styles in our app, then don't do anything.